GLMakie/src/GLAbstraction/GLRender.jl

```diff
@@ -10,10 +10,10 @@ function setup_clip_planes(robj)
     for i in 0:min(7, N-1)
         glEnable(GL_CLIP_DISTANCE0 + UInt32(i))
     end
-    # TODO: This crashes?
-    # for i in N:7
-    #     glDisable(GL_CLIP_DISTANCE0 + UInt32(i))
-    # end
+    # TODO: This crashes? But is also necessary...
+    for i in max(0, N):7
+        glDisable(GL_CLIP_DISTANCE0 + UInt32(i))
+    end
```

Comment:
Not sure in what context this was crashing but it's not crashing anymore on my laptop or PC. 
Without this turning clip_planes on once will turn them on forever, with random data. This causes plots to disappear randomly. Maybe this should be a hotfix instead of being part of this pr...